admin_ui_support/tests/src/Functional/RedirectRoutesTest.php

```diff
@@ -40,15 +52,22 @@ public function testRouteRedirect() {
     $paths = [
       '/admin/people/permissions',
       '/admin/people/roles',
+      '/node/add',
+      '/node/add/page',
+      '/node/1/edit',
     ];
 
     /** @var \Drupal\Core\Routing\RouteProvider $route_provider */
     $route_provider = \Drupal::service('router.route_provider');
     foreach ($paths as $path) {
       $routes = $route_provider->getRoutesByPattern($path);
-      $this->assertEquals(1, $routes->count());
-      foreach ($routes as $route) {
-        $this->assertEquals('Drupal\admin_ui_support\Controller\DefaultController::getAppRoute', $route->getDefault('_controller'));
+      $this->assertGreaterThanOrEqual(1, $routes->count());
+      foreach ($routes as $route_name => $route) {
+        // The canonical node route /node/1 is a route suggestion for
+        // /node/1/edit.
```

Comment:
Should we just test for this in an `else`

Reply:
What do you mean with that. Have a special test for the `/node/` routes?